python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_isis.schema.yml

```
@@ -164,6 +164,7 @@ keys:
                       - sha-512
               rx_disabled:
                 type: bool
+                description: Disable authentication on receive side.
```

ref https://www.arista.com/en/support/toi/eos-4-22-1f/14252-disabling-the-is-is-authentication-check-on-the-receive-side
```suggestion
                description: Disable authentication check on the receive side.
```

---

same for all other occurrences.